view/adminhtml/templates/tracking.phtml

```diff
@@ -1,11 +1,12 @@
 <?php
+/** @var  \Magento\Backend\Block\Template $block */
 /** @var \Algolia\AlgoliaSearch\ViewModel\Adminhtml\Common $viewModel */
-$view = $this->getViewModel();
+$view = $block->getViewModel();
 $applicationId = $view->getApplicationId();
 ?>
 
 <script>
-var applicationId = <?php echo json_encode($applicationId); ?>;
+var algoliaAppId = '<?php echo $block->escapeHtmlAttr($applicationId) ?>';
```

Comment:
@bsuravech, right escape function is $block->escapeJs($applicationId), escapeHtmlAttr uses for escape value in html attrbutes

Reply:
ah word! thanks!